modules/Users/authentication/SAML2Authenticate/lib/onelogin/php-saml/composer.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Remove this file for the time being - we'll move this to the main composer file in upcoming releases.